deploy-cluster-aws/fabfile.py

```diff
@@ -97,45 +97,94 @@ def upgrade_setuptools():
     sudo('pip3 install --upgrade setuptools')
 
 
-# Install RethinkDB
+# Prepare RethinkDB storage
 @task
 @parallel
-def install_rethinkdb():
-    """Installation of RethinkDB"""
-    with settings(warn_only=True):
-        # preparing filesystem
-        sudo("mkdir -p /data")
-        # Locally mounted storage (m3.2xlarge, but also c3.xxx)
+def prep_rethinkdb_storage(USING_EBS):
+    """Prepare RethinkDB storage"""
+    # Convert USING_EBS from a string to a bool
+    USING_EBS = (USING_EBS.lower() == 'true')
+
+    # Make the /data directory for RethinkDB data
+    sudo("mkdir -p /data")
+
+    # OLD: with settings(warn_only=True):
+    if USING_EBS:  # on /dev/xvdp
+        # See https://tinyurl.com/h2nut68
+        sudo("mkfs -t ext4 /dev/xvdp")
```

Comment:
Yeah, when the instance is launched by `launch_ec2_nodes.py`, an associated EBS volume is also launched, and I set it to have the "p" device name (on the line `'DeviceName': '/dev/sdp',`).

From the comment:  Why /dev/sdp? See https://tinyurl.com/z2zqm6n